config/routes.yaml

```diff
@@ -1,3 +1,3 @@
-#index:
-#    path: /
-#    controller: WMDE\BannerServer\Controller\DefaultController::index
+banner_server_provide:
+  path: /provide
```

Comment:
"Provide" is very generic.Do you want to avoid the word "banner"? I think the "action sentence" schema we agreed upon might be a good fit.

Or you could simulate a resource by naming the route `/banner.js`.